src/RootComponent.js

```diff
@@ -1,9 +1,6 @@
 import React, { Component } from 'react';
-import { connect } from 'react-redux';
-import PropTypes from 'prop-types';
-import { SearchBar } from './Recepies/SearchBar/SearchBar';
-import { SearchResults } from './Recepies/SearchResults/SearchResults';
-import { simpleAction } from './actions/simpleAction';
+import SearchBar from './Recepies/SearchBar/SearchBar';
```

Comment:
make changes to the way you're exporting `SearchBar` and `SearchResult(s)` from `/Recipes` so that it's possible to import them like so:
```JavaScript
import SearchBar from './Recepies';
import SearchResult from './Recepies';
```

Also note the typo in your directory's name: **Recepies**

Reply:
I have also tried to change the way that I am importing searchBar and SearchResults.